src/content/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform.mdx

```diff
@@ -0,0 +1,216 @@
+* Entity-specific tracing. You can also go to a specific monitored entity (for example, an APM-monitored app) and click **Distributed tracing** there. That shows you traces that involve that specific entity. 
+
+<img
+  title="Distributed tracing trace waterfall"
+  alt="Distributed tracing trace waterfall"
+  src={traceWaterfall}
+/>
+### Service maps [#maps]
+
+When you're in the UI for a specific monitored entity, you can click **Service map** to see a visual map of how that service relates to other monitored entities. For more details, see [Service maps](/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/introduction-service-maps).
+
+<img
+  title="APM service map"
+  alt="APM service map"
+  src={serviceMap}
+/>
+
+### Workloads [#workloads]
+
+Our [workloads](/docs/new-relic-solutions/new-relic-one/workloads/workloads-isolate-resolve-incidents-faster) feature lets you group together related entities in ways that make sense to your business. This becomes more and more important the bigger your organization is. Here's one of the UI pages you can find in a workload: a map showing the workload entities. 
+
+<img
+  title="Workload map"
+  alt="Workload map"
+  src={workloadMap}
+/>
+
+## Query your data [#query]
+
+<img
+  title="nrql_example.png"
+  alt="nrql_example.png"
+  src={queriesnrqlNrqlExampleTimeseries}
+/>
+
+Learning how to query your data is hugely important to getting the most out of New Relic. We give you a lot of out-of-the-box UI experiences, but to optimize your observability endeavors you'll need to bring in some custom data and make some custom dashboards. 
+
+You don't have to learn NRQL yet. Once you're feeling comfortable with New Relic and are ready to unlock more of its power, you can start delving into [our NRQL tutorials](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language).
+
+Also, you can explore your data without using NRQL. Go to the [**Metrics and events** explorer](https://one.newrelic.com/data-explorer) to get a visual interface for understanding the data you've got reporting. 
+
+## Account settings [#admin]
+
+<img
+  title="the location of the New Relic user menu"
+  alt="the location of the New Relic user menu"
+  src={accountsUserMenuLocation}
+/>
+
+In the lower left corner, you've got your user menu, which gives you access to more administrative options, including options for managing your data and your users. For docs on the options available there, see [Account settings](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings).
+
+## News and updates [#whats-new]
+
+Also, near the user menu, where it says **Help**, you can access the [`What's new` posts](https://docs.newrelic.com/whats-new). These are announcements about exciting new features and important changes. 
```

Comment:
I've never linked to a what's new before in the docs, is the link formatting correct here?